src/main/java/com/alkemy/ong/service/UserService.java

```diff
@@ -1,18 +1,65 @@
 package com.alkemy.ong.service;
 
 
-import com.alkemy.ong.dto.UserDTO;
+import com.alkemy.ong.dto.UserDto;
 import com.alkemy.ong.entity.UserEntity;
+import com.alkemy.ong.mapper.UserMapper;
+import com.alkemy.ong.repository.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
 
 import java.util.List;
 
-public interface UserService {
+@Service
+public class UserService {
 
-    List<UserDTO> getAllUser();
+    @Autowired
+    private UserRepository userRepository;
 
-    void deleteUserById(String id);
+    public UserService() {
+    }
+    //Soft deletion method for the User
 
-    UserEntity updateUser(UserEntity entity) ;
+    public void deleteUserById(String id) throws Exception {
+        userRepository.deleteById(id);
+    }
+    //Get all Users from Database.
```

Comment:
The service to get a list of users is correct

Reply:
thanks for reading